commons/src/test/java/io/aiven/kafka/tieredstorage/commons/UniversalRemoteStorageManagerTest.java

```diff
@@ -364,4 +370,42 @@ private void checkDeletion() throws RemoteStorageException {
         rsm.deleteLogSegmentData(REMOTE_LOG_METADATA);
         assertThat(targetDir).isEmptyDirectory();
     }
+
+    @ParameterizedTest
+    @CsvSource({"NONE,true", "ZSTD,false"})
+    void testRequiresCompression(final CompressionType compressionType, final boolean result)
```

Comment:
[Nit] I'd probably rename like this
```suggestion
    void testRequiresCompression(final CompressionType compressionType, final boolean expectedResult)
```